### PR TITLE
Created partials for displaying matches, players and player form

### PR DESCRIPTION
**Pull request by chesterhow**
@moexmen/interns

### CODE REVIEW

**Comment by abeltay** [comment]
Can you rename your stylesheets from `*.css.scss` to `*.scss`?

File: app/views/matches/index.html.slim [hunk 1]
```diff
@@ -4,93 +4,11 @@
...
-- @matches.includes(:player_matches, :players).each do |match|
-  table.match-tbl
-    thead
-      tr
-        - if @date_var != pretty_format_date(match.created_at)
-          - @date_var = pretty_format_date(match.created_at)
-          th[colspan="5"]
-            = @date_var
-
+- @matches.includes(:player_matches, :players).reverse_each do |match|
```
Comment (sohymg) [inline app/views/matches/index.html.slim:7]: more rails magic (can do next time)
`= render @matches.includes(:player_matches, :players).reverse`

then in `_match.html.slim` you can access the variable `match`

check out section 3.4.5 in http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials

File: app/views/players/edit.html.slim [hunk 2]
```diff
@@ -2,16 +2,7 @@ h1.title
   | EDIT PLAYER
 .form
   = form_for :player, url: player_path(@player), method: :patch do |f|
```
Comment (weiqingtoh) [inline app/views/players/edit.html.slim]: Actually you can just put this line (and the line above with `.form`) into the form partial, and just pass the `@player` variable into that partial. So you don't have to pass a form object around. 

That will allow you to reuse the entire form on other pages if you need to next time.

Comment (chesterhow) [inline app/views/players/edit.html.slim]: @weiqingtoh The `form_for` for edit is 

```
= form_for :player, url: player_path(@player), method: :patch do |f|
```

and the `form_for` for new is 

```
= form_for :player, url: players_path do |f|
```

So I'm not so sure if i can pass the `@player` variable to both.

Comment (weiqingtoh) [inline app/views/players/edit.html.slim]: You can pass variables into the partial: `= render 'form', locals: { player: @player }`

Then in your `_form.html.slim`:

``` slim
= form_for player do |f|
  .row
    span.form-label
      ...
```

Actually if you pass in the correct player, rails will try to deduce the url and HTTP method for you. You can read up at [this link](http://apidock.com/rails/ActionView/Helpers/FormHelper/form_for), search for 'Resource-oriented style'. 

cc. @moexmen/interns - For learning as well :+1:

**Pull request merged**